src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -335,7 +335,7 @@ export class StudySummaryTab extends React.Component<IStudySummaryTabProps, {}>
                                         onChange={this.props.store.toggleWithMutationDataFilter}
                                         disabled={this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined}
                                     >
-                                        {this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined ? '0' : this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples.toLocaleString()} w/ mutation data
+                                        {this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined ? '0 sample' : this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples.toLocaleString() + (this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples <= 1 ? " sample" : " samples")} with mutation data
```

Comment:
this line is so long, can we save `const numMutationProfiledSamples = this.props.....numberOfMutationProfiledSamples` and reuse it